Makefile

```diff
@@ -9,10 +9,10 @@ lint:
 	mypy
 	@echo "🙊 Code 🙈 LGTM 🙉 !"
 
-tests/scripts/config/pulp/settings.toml:
+tests/settings.toml:
 	cp $@.example $@
 
-test: | tests/scripts/config/pulp/settings.toml
+test: | tests/settings.toml
```

Comment:
This move is a minor inconvenience to current developers. But it makes this file so much more accessible to new contributors, that i think it is worth the trouble.